backend/api.py

```diff
@@ -1 +1,11 @@
 # TODO: Add basic FastAPI app for communicating with the front end
+
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.get("/health")
+async def health_check():
+    return {"message": "Infer GPT is running!"}
```

Comment:
v. minor question, but will pep-8 implement whitespacing rules?

Reply:
We can configure it not to if it's annoying, but to start I think it will

Reply:
It's not set up yet however so one to re-look at this when the time comes

Reply:
yeah I'm keen for it to enforce consistency when it is set up, cool thanks will approve